src/api/student.ts

```diff
@@ -187,6 +187,15 @@ router.get('/degrees', async (req: Request, res: Response) => {
         req.session.passport.user = user;
       }
     }
+
+    // if user's degree contains "in", get rid of it
+    if (response.data.length > 0) {
+      if (response.data[0].attributes.degree.includes(' in')) {
```

Comment:
We should check if degree endsWith ' in' and replace only that last occurrence if it exists.